test/e2e/federated-namespace.go

```diff
@@ -97,7 +94,23 @@ var _ = framework.KubeDescribe("Federation namespace [Feature:Federation]", func
 	})
 })
 
-func deleteAllTestNamespaces(clientset *federation_release_1_4.Clientset) {
+func deleteAllTestNamespacesFed14(clientset *federation_release_1_4.Clientset) {
```

Comment:
Hmm - this is so weird that we have two interfaces for the same thing that are a bit different.
@quinton-hoole ^^

Anyway - let's merge as is for now.